modules/network-observability-without-loki.adoc

```diff
@@ -0,0 +1,19 @@
+// module included in the following assemblies:
+// networking/network_observability/installing-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-without-loki_{context}"]
+= Network Observability without Loki
+You can use Network Observability without Loki by omitting the Loki installation steps and only using exporters, such as Kafka or IPFIX. The following table compares available features with and without Loki:
```

Comment:
"Only" is a tricky word to place. Maybe after the verb is less restrcitive?

> and by using only exporters to X

Reply:
s/by omitting the Loki installation steps/not performing the Loki installation steps in the following section